python/mxnet/numpy/multiarray.py

```diff
@@ -40,8 +40,9 @@
                       get_oshape_of_gather_nd_op
 from ..ndarray._internal import _set_np_ndarray_class
 from . import _op as _mx_np_op
-from ..base import check_call, _LIB, NDArrayHandle, c_array
+from ..base import check_call, _LIB, NDArrayHandle, c_array, mx_int, mx_int64
```

Comment:
This is actually correct no need to change it.

Reply:
mx_int/mx_int64 are used below.